app/graphql/resolvers/password_reset_resolver.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class PasswordResetResolver < Resolvers::BaseResolver
```

Comment:
Do we need this resolver, where is it used? 🤔

Reply:
Yeah we need it on the FE side.

It's called when we access the reset password page to prevent using an expired password reset token